### DISCUSSION TITLE
How to customize the title name for the table？

### DISCUSSION THREAD

**Discussion by junmaqiang** [fyne-io/fyne#4227] [Q&A]
# I want to replace ABCDEF with a custom title name. How do I do this?

<img width="816" alt="截屏2023-09-10 08 54 41" src="https://github.com/fyne-io/fyne/assets/1819457/0908daa3-62bc-4b36-9064-75299c62b1de">

**Comment by junmaqiang** [comment]
Before upgrading to 2.4, the table did not have a header row. I customized one. Now I want to use the header row that comes with the table, but there is no reference use case.

**Reply by andydotxyz** [reply]
Set the `UpdateHeader` callback (it works like `UpdateCell` https://developer.fyne.io/api/v2.4/widget/table.html

**Comment by junmaqiang** [comment]
The new version of the table list item will be temporarily stuck after being selected.

**Reply by andydotxyz** [reply]
Please don't use discussions to report a bug, open an issue.

**Comment by junmaqiang** [comment]
There is a bug in the table component. Sometimes if you accidentally touch the unit, it will get stuck.
